src/backend/api/handlers.py

```diff
@@ -213,14 +213,29 @@ async def _process_mirror(
 
 
 async def update_mirrors_handler() -> str:
-    config = get_config()
+    config = get_config(
+        logger=logger,
+        path_to_config=os.path.join(
+            os.getenv('CONFIG_ROOT'),
+            'mirrors/updates/config.yml'
+        ),
+        path_to_json_schema=os.path.join(
+            os.environ['SOURCE_PATH'],
+            'src/backend/yaml_snippets/json_schemas/service_config.json'
+        )
+    )
     mirrors_dir = os.path.join(
         os.getenv('CONFIG_ROOT'),
         'mirrors/updates',
         config.mirrors_dir,
     )
     all_mirrors = get_mirrors_info(
         mirrors_dir=mirrors_dir,
+        logger=logger,
+        path_to_json_schema=os.path.join(
+            os.environ['SOURCE_PATH'],
+            'src/backend/yaml_snippets/json_schemas/mirror_config.json',
```

Comment:
Well, I saw the constant 4 times. If I were you I would use the constant instead of literal